openid-caep-interoperability-profile-1_0.md

```diff
@@ -375,6 +380,21 @@ generate any allowable value of this field
 `reason_admin`
 : Transmitters MUST populate this value with a non-empty string
 
+## Device Compliance Change
+
+In order to support notifying and responding to changes in device compliance status, implementations MUST support the event type `device-compliance-change`. This event is used to signal that a device's adherence to a set of security or organizational compliance policies has changed. implementations MUST support the following field values:
```

Comment:
this line is too long... can you fix the formatting so the line breaks are correct.
The linter should also catch this.

Reply:
won't the kramdown / xml2rfc tools take care of the line length?